### PR TITLE
Speedrun goals for 10/7 to 10/13

### PR DESCRIPTION
**Pull request by alp203**

### CODE REVIEW

File: management/SpeedRun.md [hunk 1]
```diff
@@ -28,18 +28,17 @@
 1. 9/30 - 10/6
 	* Manager: Connor Kalina
 	* Goals:
-		1. Modify the game loop to incorporate networked multiplayer without affecting single-player, while avoiding multithreading
+		1. Networking goal for next week will be making things multithreaded so the client can connect to the server while the game is playing
```
Comment (nfarnan) [inline management/SpeedRun.md:31]: You'll need to rebase your version of the organization repo and recreate this pull request. I do not want the prior week's goals changed.

**Review by nfarnan — CHANGES_REQUESTED** [review]
Other than modifying the approved goals for the previous week, this looks OK

**Comment by nfarnan** [comment]
Issue a new request after rebasing.

File: management/SpeedRun.md (hunk 1 above)
Comment (alp203) [reply management/SpeedRun.md:31]: No idea how this happened, I think I had an older version from before we changed course for last week. I will fix it.

**Pull request closed without merging**